### DISCUSSION TITLE
How to use Laravel API Resources with Inertia?

### DISCUSSION THREAD

**Discussion by AlexKomMsk** [inertiajs/inertia#1302] [Help (Vue)] [answered]
I am using Laravel 9, Jetstream, Inertia, Vue 3.

I have created an Api Resource for my Model Project

```
$projects = ProjectResource::collection(Project::get());

return Inertia::render('Project/Edit', compact('projects'));
```

In Vue, in the props "project" i get a nested array "data" and only the objects I need are already in it.

```
projects:
data:Array[2]
0:Object
1:Object
```

But this is wrong. I don't need a nested data array. I immediately need models in the prop, how it is transferred to the view without using Inertia. Those it should be like this

```
projects:
0:Object
1:Object
```

I don't want to access props in vue via "projects.data"
I want it to be: "projects"
How to achieve this?

**Comment by AlexKomMsk** [comment, marked as answer]
Found the answer in the documentation https://laravel.com/docs/9.x/eloquent-resources#data-wrapping
Weird, I haven't had to change the AppServiceProvider for this before..

**Reply by darthsoup** [reply]
Note that there is currently a bug that sometimes prevents unpacking. https://github.com/inertiajs/inertia-laravel/pull/432
